src/UI/Layout/SplitSelectable.elm

```diff
@@ -21,8 +21,8 @@ type alias Config object msg =
 
 
 type alias MobileConfig msg =
-    { buttons : List (Button msg)
-    , title : String
+    { rightButton : Maybe (Button msg)
```

Comment:
why was this change necessary? Are we not supplying more than one button in some cases?

Reply:
I didn't found any case. And when there are no buttons, Sufyan centers the title as if there was one invisible button in the right (or ignores the button on left for alignment).